pkg/k8s/forward/manager.go

```diff
@@ -146,7 +146,9 @@ func (p *PortForwardManager) Start(devPod, namespace string) error {
 			if !errors.Is(err, portforward.ErrLostConnectionToPod) {
 				p.activeDev.closeReady()
 			}
-			p.activeDev.err = err
+			if p.activeDev != nil {
```

Comment:
Should this check also apply to line 147? We are using also `activeDev`. Is activeDev != nil always in that scenario?

Reply:
It shouldn't be needed but adding it in case there is a corner case I'm not contemplating